res/xml/urdu phonetic

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="Urdu phonetic" script="urdu">
+  <row>
+    <key key0="ق" key2="1" key4="esc"/>
+    <key key0="و" key2="2" key4="redo"/>
+    <key key0="ع" key1="!" key2="3"/>
+    <key key0="ر" key2="ڑ" key4="cut"/>
```

Comment:
The `cut`, `selectAll`, `copy`, etc.. shouldn't be added to layouts. No other layout contain them.

I'm open to discuss ways to add these keys later.

Reply:
I did this so that these buttons remains consistent in every layout because these button get randomly assigned when added from the add keys option.